specs/core/0_beacon-chain.md

```diff
@@ -653,57 +653,73 @@ DEPOSIT_CONTRACT_TREE_DEPTH: constant(uint256) = 32
 TWO_TO_POWER_OF_TREE_DEPTH: constant(uint256) = 4294967296  # 2**32
 SECONDS_PER_DAY: constant(uint256) = 86400
 
-Deposit: event({previous_deposit_root: bytes32, data: bytes[2064], merkle_tree_index: bytes[8]})
+Deposit: event({previous_deposit_root: bytes32, data: bytes[2064], merkle_tree_index: bytes[8], branch: bytes32[32]})
```

Comment:
Should we rename `previous_deposit_root` to `deposit_root`? or we still need to log previous deposit root?

Reply:
Yes, in this version the argument is not the previous `root`. 
If indeed the current `root` is wanted here then it can be removed at all, since it can be calculated from the `branch`